vue/components/ui/molecules/button-dropdown/button-dropdown.vue

```diff
@@ -192,7 +208,20 @@ export const ButtonDropdown = {
                 disabled: this.disabled
             };
             base["button-dropdown-" + this.size] = true;
+            base["button-dropdown-" + this.color] = true;
             return base;
+        },
+        buttonColorPrimary() {
+            if (this.color === "black" && !this.primaryActive) return "#ffffff";
+            if (this.color === "black" && this.primaryActive) return "#000000";
+            if (this.primaryActive) return "#ffffff";
+            return "#000000";
+        },
+        buttonColorSecondary() {
+            if (this.color === "black" && !this.secondaryActive) return "#ffffff";
+            if (this.color === "black" && this.secondaryActive) return "#000000";
+            if (this.secondaryActive) return "#ffffff";
+            return "#000000";

```

Comment:
The conditions here a little to redundant. You can reduce the number of checks

Reply:
I believe this is the most readable one, but feel free to suggest something else

Reply:
```js
if (primaryActive) return color === "black" ? "#000000" : "#ffffff";
return color === "black" ? "#ffffff" : "#000000";
```

And the `buttonColorSecondary` one is similar to this code block

Reply:
In my opinion this one is less readable but more concise. I'll use it!